posts/indu-plp-part2/plp-part2.qmd

```diff
@@ -0,0 +1,281 @@
+---
+title: "PLP-Pipeline Series Part 2: From Raw Clinical Data to Predictive Models"
+description: "Part 2 of the PLP-Pipeline blog series – how we preprocess OMOP CDM data, extract features, and train ML models using Julia tools"
+author: "Kosuri Lakshmi Indu"
+date: "4/20/2025"
+bibliography: ./references.bib
+csl: ./../../ieee-with-url.csl
+toc: true
+engine: julia
+image: false
+categories:
+  - patient-level prediction
+  - omop cdm
+  - observational health
+  - machine learning
+---
+
+# Introduction 👋
```

Comment:
I think you should also mention at the end of this section the steps you are going to take here or that you are guiding the reader through a prediction pipeline.

Reply:
Nah, I think if you have done those other edits, thinking again, I think this is perfectly fine now.